src/ripple/app/tests/MultiSign.test.cpp

```diff
@@ -172,16 +172,35 @@ class MultiSign_test : public beast::unit_test::suite
         env.close();
         expect (env.seq(alice) == aliceSeq);
 
-        // Multisign, but leave a nonempty sfSigners.  Should fail.
+        // Multisign, but leave a nonempty sfSigningPubKey.  Should fail.
         {
             aliceSeq = env.seq (alice);
             Json::Value multiSig =
                 env.json (noop (alice), msig(bogie), fee(2 * baseFee));
-
             env (env.jt (multiSig), ter (temINVALID));
             env.close();
             expect (env.seq(alice) == aliceSeq);
         }
+        // Attach both Signers and a TxnSignature with an empty sfPubKey.
+        // Should fail.
+        {
+            aliceSeq = env.seq (alice);
+            Json::Value multiSig = env.json (noop (alice),
+                fee(2 * baseFee), seq(aliceSeq), sig(alice), msig(bogie));
+            JTx jt = env.jt (multiSig);
+            jt.fill_fee = false;
+            jt.fill_seq = false;
+            jt.fill_sig = false;
+            jt.jv["SigningPubKey"] = "";
+
+            auto noSigPubKey = std::make_unique<STTx>(*(jt.stx));
+            noSigPubKey->setFieldVL (sfSigningPubKey, Blob());
```

Comment:
Great question.  A `JTx` object contains both a `Json::Value` and an `STTx` which represent the same data.  Since I'm mangling the `JTx` in ways that may not have been anticipated, I'm mangling both the `Json::Value` and the `STTx` identically.  At least that's the intention.  So I do the work twice.